src/Renderer/LayeredMaterial.js

```diff
@@ -76,7 +90,7 @@ var moveElementsArraySafe = function moveElementsArraySafe(array,index, howMany,
 };
 /* eslint-enable */
 
-const LayeredMaterial = function LayeredMaterial(id) {
+const LayeredMaterial = function LayeredMaterial(material, coord) {
```

Comment:
I disagree :)
But that something we can fix later, so I won't block any longer on this point.